src/elements/Button/Button.styl

```diff
@@ -123,17 +138,16 @@
   transition-duration $duration-shortest
 
   // Children
-
```

Comment:
That newline was important. Comments that apply to more than one block following them are separated from the first block using a newline. Comments that apply just to the following block are not separated from that block by a newline.

Reply:
// Children

& > .content
padding $space-zero